### ISSUE TITLE
A way to stop showing UI on hover over?

### ISSUE THREAD

**Issue by o4zloiroman** [ripose-jp/Memento#41]
It's rather jarring to see the seeking menu pull up every time I'm trying to hover over a word.

**Comment by o4zloiroman** [comment]
Also, the "Subtitle Visible/Hidden" prompt that appears on the V press cover part of the secondary subtitle, which is counter-intuitive, given that's the solution you provide to be able to get secondary sub to appear.

**Comment by ripose-jp** [comment]
>It's rather jarring to see the seeking menu pull up every time I'm trying to hover over a word.

That's the way the interface behaves, there is no way to change it. In version v0.5.0, the UI was fixed in windowed mode so that the OSD was always on screen, but v0.5.1 changed it to the way it currently is. I also wouldn't recommend downgrading because there are some really bad issues in that version (#34) that I'm not going to fix.

How would you like it to work? Having controls appear when the cursor is moved is fairly standard among modern players (mpv, IINA, YouTube, Twitch, etc.). I don't see any more intuitive way to do it.


>Also, the "Subtitle Visible/Hidden" prompt that appears on the V press covers part of the secondary subtitle, which is counter-intuitive, given that's the solution you provide to be able to get secondary sub to appear.

Here's a hack you can do:
1. Open Memento
2. Go to Settings > Open Config Folder
3. Create a new folder called `scripts` in the config folder
4. Create a new file called `subvis.lua` and paste this into it
```
mp.add_key_binding("v", "sub-visibility", function()
        vis = mp.get_property_bool("sub-visibility")
        mp.set_property_bool("sub-visibility", not vis)
end)
```
5. Restart Memento

**Comment by Ulidtsoa01** [comment]
Is there a way to keep the UI always visible?

**Comment by ripose-jp** [comment]
No.

Keeping the UI always visible in windowed mode, while having it disappear in fullscreen mode made the code more complicated and my implementation was buggy. I threw it out for the disappearing UI because it made things simpler and there aren't really any problems with it as far as I can tell.

**Comment by mansuen** [comment]
Couldn't the OSC be kept visible with this setting?

https://mpv.io/manual/master/#on-screen-controller-visibility

I don't know how to implement the settings though.

**Comment by ripose-jp** [comment]
This was already solved in #95. Press the **delete** key.

As for why this key is not rebindable, mpv's API is flawed (in my opinion) because if `osc=no` is set, none of the other OSC settings are tracked, including visibility. Memento can't use `osc=yes` because it uses its own OSC, so setting `osc=yes` makes mpv's OSC visible along side Memento's which is pointless at best and annoying at worst.
